hw6/task2/task2.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,83 @@
+using System.ComponentModel;
+
+namespace task2.Tests
+{
+    public class Tests
+    {
+        private static readonly string testFiles = "../../../TestFiles/";
+
+        [SetUp]
+        public void Setup()
+        {
+        }
+        private char[,] LoadMap(string pathToMap)
+        {
+            int maxLineLength = 0;
+            string[] lines = File.ReadAllLines(pathToMap);
+            for (int i = 0; i < lines.Length; i++)
+            {
+                maxLineLength = maxLineLength < lines[i].Length ? lines[i].Length : maxLineLength;
+            }
+            char [,] map = new char[lines.Length, maxLineLength];
```

Comment:
```suggestion
            var map = new char[lines.Length, maxLineLength];
```
Тем более что перед `[` пробел не ставится